docs/guides/player-workflows.md

```diff
@@ -0,0 +1,58 @@
+# Player Workflows
+
```

Comment:
Should this file contain other advanced workflows? like using `ready` or maybe using other events on the player to do things.